bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/discovery/HueBridgeMDNSDiscoveryParticipant.java

```diff
@@ -104,29 +112,64 @@ public String getServiceType() {
     public @Nullable DiscoveryResult createResult(ServiceInfo service) {
         if (isAutoDiscoveryEnabled) {
             ThingUID uid = getThingUID(service);
-            if (uid != null) {
+            if (Objects.nonNull(uid)) {
```

Comment:
For my curiosity, what's the benefit of this change ?

Reply:
> what's the benefit of using Objects.nonNull compared to != null

I think there is no 'benefit' as such. It is just another way of writing the same thing. And apparently the Java compiler produces the same machine code for either..

Reply:
Ok